tests/test_parser.py

```diff
@@ -258,6 +258,7 @@ def test_convert_to_date_cast_result_to_date(self, mocked_locale,
         self.assertEqual(convert_to_date_from_russian_written('', ''),
                          expected_output)
 
+
 @patch('bs4.BeautifulSoup')
```

Comment:
You don't need to mock `BeautifulSoup` here as the only thing you use from it is `select` method. Create new mock with `select` method that returns what you need.

Reply:
I can’t mock @patch('bs4.BeautifulSoup.select') in Class because I use here 
soup = mocked_soup('some html', 'html.parser') bs4.BeautifulSoup mocking